.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,59 @@
+#
+name: Create and publish a Docker image
+
+on:
+  release:
+    types: [published]
```

Comment:
I would like to build the "latest" on every PR merge event. Can you add it here as well?

Reply:
Done. It will build a new release on a push to the main branch which covers PR merges as well as direct main commits.